show-land/sass/main.scss

```diff
@@ -0,0 +1,6 @@
+@import"variables";
+@import"base";
+@import"buttom";
+@import"nav";
+@import"typography";
+@import"../../shared/scss/base/_reset.scss";
```

Comment:
you need to use the base in the first line, and then import others, the priority of using the files is important. 
as I said before, you can remove base from your project because we already defined on shared folder